src/components/forms/file-dropper/file-dropper.tsx

```diff
@@ -197,7 +208,9 @@ export const FileDropper = ({
         hasFiles={hasFiles}
         hasErrors={errors?.documents}
       >
-        <input {...getInputProps({ ...register(name) })} />
+        <StyledLabel>
+          <input {...getInputProps({ ...register(name) })} />
+        </StyledLabel>
```

Comment:
Do you know why that changes the behavior? I mean I understand the thing behind label + input but why does it not work without? How did you came to this fix?

Reply:
I found out that there can be problems with file inputs in combination with `diaplay: none`. The solution with the label resulted from trial and error